File: 1627-last-moment-before-all-ants-fall-out-of-a-plank/last-moment-before-all-ants-fall-out-of-a-plank.js
/**
 * @param {number} n
 * @param {number[]} left
 * @param {number[]} right
 * @return {number}
 */
var getLastMoment = function(n, left, right) {
    let max = -1;
    for(val of left) max = Math.max(max, val);
    for(val of right) max = Math.max(max, n - val);
    return max;
};

File: 1627-last-moment-before-all-ants-fall-out-of-a-plank/last-moment-before-all-ants-fall-out-of-a-plank.java
class Solution {
    public int getLastMoment(int n, int[] left, int[] right) {
        int max = -1;
        for(int val : left) max = Math.max(max, val);
        for(int val : right) max = Math.max(max, n-val);
        return max;
    }
}